models/product-data/README.md

```diff
@@ -0,0 +1,20 @@
+# @commercetools-test-data/product-data
+
+This package provides the data model for the commercetools platform `ProductData` type
+
+https://docs.commercetools.com/api/projects/products#productdata
+
+# Install
+
+```bash
+$ yarn add -D @commercetools-test-data/product-data
+```
+
+# Usage
+
+```ts
+import type { TProductData } from '@commercetools-test-data/product-data';
+import * as ProductData from '@commercetools-test-data/product-data';
+
+const productData = ProductData.random().build<TProductData>();
```

Comment:
Given the new types, is this the correct usage?

Reply:
Adjusted in 47033449c4547cbc1ac2cf003703b877bb73e07c